app/components/primer/beta/flash.rb

```diff
@@ -42,29 +36,27 @@ class Flash < Primer::Component
       # @example With actions
       #   <%= render(Primer::Beta::Flash.new) do |component| %>
       #     This is a flash message with actions!
-      #     <% component.with_action do %>
-      #       <%= render(Primer::ButtonComponent.new(size: :small)) { "Take action" } %>
-      #     <% end %>
+      #     <% component.with_action(size: :small) { "Take action" } %>
       #   <% end %>
       #
       # @param full [Boolean] Whether the component should take up the full width of the screen.
-      # @param spacious [Boolean] Whether to add margin to the bottom of the component.
+      # @param full_when_narrow [Boolean] Whether the component should take up the full width of the screen when rendered inside smaller viewports.
       # @param dismissible [Boolean] Whether the component can be dismissed with an X button.
       # @param icon [Symbol] Name of Octicon icon to use.
       # @param scheme [Symbol] <%= one_of(Primer::Beta::Flash::SCHEME_MAPPINGS.keys) %>
       # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-      def initialize(full: false, spacious: false, dismissible: false, icon: nil, scheme: DEFAULT_SCHEME, **system_arguments)
```

Comment:
We'll need some updates to code in production on github for this.

Reply:
Added a linter.